code/game/gamemodes/gameticker.dm

```diff
@@ -134,7 +134,10 @@ var/round_start_time = 0
 	else
 		src.mode.announce()
 
+	setupfactions()
+	setup_economy()

```

Comment:
Wait....

Reply:
That jump list isnt working, can I get file and line? And if it was doing it twice, maybe thats why we are so slow to start

Reply:
Diagnosis: I cant raed